crates/configuration/src/relaychain.rs

```diff
@@ -135,11 +141,11 @@ impl<A> RelaychainConfigBuilder<A> {
 
     fn default_chain_context(&self) -> ChainDefaultContext {
         ChainDefaultContext {
-            default_command: self.config.default_command.clone(),
-            default_image: self.config.default_image.clone(),
+            default_command: self.config.default_command().cloned(),
+            default_image: self.config.default_image().cloned(),
```

Comment:
Not needed, the getter are for the public API, we have access to the inner directly here:

```suggestion
            default_command: self.config.default_command.clone(),
            default_image: self.config.default_image.clone(),
```